crypto/asn1/charmap.pl

```diff
@@ -82,12 +82,14 @@
 
 # Now generate the C code
 
+my @lt = localtime;
+my $YEAR = $lt[5] + 1900;
```

Comment:
If you want to avoid have `@lt`, you can do this:

``` perl
my $YEAR = [ localtime() ]->[5] + 1900;
```

Reply:
Which is better for modern perl monks?  (I am an old perl initiate.)